modules/utxo-lib/src/bitgo/outputScripts.ts

```diff
@@ -35,6 +35,22 @@ export type SpendableScript = {
   controlBlocks?: [userBitGoScript: Buffer, userBackupScript: Buffer, backupBitGoScript: Buffer];
 };
 
+/**
+ * Return scripts for p2sh-p2pkh (used for BCH/BSV replay protection)
+ * @param pubkey
+ */
+export function createOutputScriptP2shP2pkh(pubkey: Buffer): SpendableScript {
+  const p2pkh = bitcoinjs.payments.p2pk({ pubkey });
```

Comment:
I'm guessing this should use `payments.p2pkh`?

Reply:
nope, I misnamed the function